crates/iroha_data_model/src/query/mod.rs

```diff
@@ -1122,31 +1122,31 @@ pub mod error {
         #[ffi_type(opaque)]
         pub enum FindError {
             /// Failed to find asset: `{0}`
-            Asset(AssetId),
+            Asset(Box<AssetId>),
             /// Failed to find asset definition: `{0}`
-            AssetDefinition(AssetDefinitionId),
+            AssetDefinition(Box<AssetDefinitionId>),
             /// Failed to find NFT: `{0}`
-            Nft(NftId),
+            Nft(Box<NftId>),
             /// Failed to find account: `{0}`
-            Account(AccountId),
+            Account(Box<AccountId>),
             /// Failed to find domain: `{0}`
-            Domain(DomainId),
+            Domain(Box<DomainId>),
             /// Failed to find metadata key: `{0}`
-            MetadataKey(Name),
+            MetadataKey(Box<Name>),
```

Comment:
Not sure that it is worth to `Box` every variant here. `FindError` is 64 bytes, and its largest variants are `FindError::Asset` (56 bytes) and `FindError::Permission` (48 bytes). All other variants are 32 bytes or less. 

Maybe add `Box` on upper level (e.g. `QueryExecutionFail::Find`), or box only largest variants (`Asset` and `Permission`)?

Reply:
Boxed only `AssetId` and `Permission`